torch/utils/data/datapipes/iter/callable.py

```diff
@@ -131,6 +136,33 @@ def __setstate__(self, state):
             self.fn = dill_function  # type: ignore[assignment]
 
 
+def _collate_helper(conversion, item):
+    # TODO(VitalyFedyunin): Verify that item is any sort of batch
+    if len(item.items) > 1:
+        print(item.items)
```

Comment:
nit: removing print

And, I am curious where does the attribute `items` come from? What is the actual type of `item`?

Reply:
`DataChunkDF`